templates/shared/forms/interactive/osm-contact-us.html

```diff
@@ -0,0 +1,254 @@
+<div class="p-modal" id="contact-modal">
+  <div class="p-modal__dialog" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description">
+    <header class="p-modal__header" style="display: block; border-bottom: 0;">
+      <button class="p-modal__close" aria-label="Close active modal" style="margin-left: -1rem">Close</button>
+      <h2 class="p-modal__title u-sv1" id="modal-title">Orchestrate your network functions</h2>
+    </header>
+
+    <div class="js-pagination js-pagination--1">
+      <p id="modal-description" class="u-no-max-width"></p>
+
+      <div class="js-formfield">
+        <h3 class="p-heading--five">What are the infrastructure preferences for your network function? (Optional)</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-kubernetes-cluster" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-kubernetes-cluster">Kubernetes cluster</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-bare-metal" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-bare-metal">Bare metal</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-public-cloud" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-public-cloud">Public cloud</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-openstack" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-openstack">OpenStack</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-vmware" name="infrastructure-preferences">
+            <label for="infrastructure-preferences-vmware">VMware</label>
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="infrastructure-preferences-other" name="infrastructure-preferences">
```

Comment:
We use a text input for other like this:
`              <input type="text" id="infrastructure-preferences-other" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other architecture">`